text/survey_analysis_report2.md

```diff
@@ -0,0 +1,173 @@
+especially as learning a new workflow can take time.
+
+
+
+##  Shortcomings of the project-level structure
+
+In many cases, people mentioned that the files are organized via experiments, not at the project level.
+In particular, people tend to pool data and code in a single experiment folder.
+This is reminiscent of the two example structure given in the library carpentry course [@LibraryCarpentryLcfairresearch2019].
+
+
+On the other hand, some files are saved in particular places irrespectively of the project they belong to. 
+Many researchers reported having a folder for all conferences reports or all manuscritps, for instance.
+
+
+## Our analysis
+
+One main rule of data management is to keep all files related to a project in a single folder,
+and we would like to follow this principle,
+because it facilitates sharing of these files with the whole team working on the project.
+On the other hand, there is a value in having files organised by type and not by projects, especially for PIs, who may only be interested to access figures or conference presentation for the different projects.
+We therefore propose to mirror these files both in the project folder and in a different folder.
+This is technically possible using the git submodule technology: the data is only once on the server, but can be provided twice on the computer. 
+Modifying one or the other version of the file will be synchronised using git technologies.
+
+On the other hand,
+the distinction between data and code is primordial for an efficient work and data management. 
+Indeed, data requires a different type of version control than code,
+it is therefore preferable to keep these different file types in different folders.
+In addition, code should be reusable on different datasets,
+it makes therefore sense to be able to share it independently of the data.
+And finally, it is easy to make code read data present in a different folder and there is no technical reason to keep data and code together.
+We will therefore keep an organization where the first level separate data and code.
+However, we want to implement a different solution that creates new folders in the different first level folders, for each new experiment. 
+
+# Gin-tonic solution
+
+The gin tonic application (thats brings tonic vigor to the g-node infrastructure gin) is still in development, 
+but will implement the different steps we presented above.
+It relies on the git, git annex, and git submodule technologies. 
+In gin-tonic, sub-folders can be submodules,
+meaning they can be synchronised, shared and published independently of the other sub-folders,
+while looking completely normal on one's computer.
+A mockup presentation for a lab with 2 projects is available at https://gin.g-node.org/testorga.
+
+![Fig.4: Folder structure developed after analysing the survey.](../50_figures/foldertree_Screenshots/foldertree.png)
```

Comment:
Another point on the image: the use of "labels" for pointing specific directories does not seem to work.
Can you make rectangles overlaid on the folder entry (i.e. the space that includes both the icon and the name)?